keyboards/handwired/stream_cheap/2x4/rules.mk

```diff
@@ -20,3 +20,4 @@ BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
+VIA_ENABLE = yes			# VIA support
```

Comment:
```suggestion
```
This goes in a separate rules.mk under the `via` keymap.

Reply:
Addressed in #4d5e4f1